python/ql/lib/semmle/python/Concepts.qll

```diff
@@ -421,6 +421,26 @@ module RegexExecution {
   }
 }
 
+/**
+ * A node that is not a regular expression literal, but is used in places that
+ * may interpret it as one. Instances of this class are typically strings that
+ * flow to method calls like `re.compile`.
```

Comment:
I find it confusing that this talks about strings that flow to places. Is the point of this refactor not, that this concept only includes those places (where strings are being interpreted as regular expressions)?
The definition of `regSink` suggest that we do tracking based on the `RegExpInterpretation` concept among other things.

Also, I think we do not have regular expression literals in Python.
```suggestion
 * A node where a string is interpreted as a regular expression,
 * for instance an argument to `re.compile`.
```

Reply:
Yes, your comment is better, and yes the concept only includes the places where a string is interpreted as a regular expression.  

> Also, I think we do not have regular expression literals in Python.

Too much copy-paste from my end.